### PR TITLE
fix(storage):move utils func from cluster operator to operator

### PR DESCRIPTION
**Pull request by soulseen**
fix https://github.com/goharbor/harbor-operator/issues/208

Signed-off-by: soulseen <zhuxiaoyang1996@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,7 +11,7 @@ require (
 	github.com/go-kit/kit v0.10.0
 	github.com/go-logr/logr v0.2.1-0.20200730175230-ee2de8da5be6
 	github.com/go-redis/redis v6.15.9+incompatible
-	github.com/goharbor/harbor-cluster-operator v0.5.0
+	github.com/goharbor/harbor-cluster-operator v0.5.0 // indirect
```
Comment (steven-zou) [inline go.mod:14]: why we still need this pkg dependency? It can be removed, right?

**Review by steven-zou — COMMENTED** [review]
has a comment

File: go.mod (hunk 1 above)
Comment (soulseen) [reply go.mod]: right, removed.

**Review by chlins — APPROVED** [review]
LGTM

**Review by Ghostbaby — APPROVED** [review]
lgtm

**Pull request merged**